src/components/checkbox/Checkbox.js

```diff
@@ -183,7 +183,7 @@ export default class CheckBoxComponent extends Field {
   }
 
   getValueAsString(value) {
-    return value ? 'Yes' : 'No';
+    return this.t(value ? 'Yes' : 'No');
```

Comment:
Should these keys be added also in https://github.com/formio/formio.js/blob/master/src/translations/en.js ?

Reply:
Thank you for your review. I agree, yes it would normally be the case, but for this case "Yes" and "No" can very easily be added to the translations like

```
{
  i18n: {
    es: {
      'Yes': 'Si',
      'No': 'No'
    }
  }
}
```